notes/transcript15.md

```diff
@@ -213,108 +213,228 @@ Igor: I guess the signature won't signatures won't match but we can do it a shot
+
+Ricardo: So it would also be possible to option to the users to opt-in this and also other protocol is not using it anymore. But of course this opt-in should be explaining the user what changes and what's why it's better, why  less privacy and etc.
+
+Oskar: So so this would be so this would be more about client implementers and then maybe it's the Status app would choose to expose such options but across I mean it's more amount of Status clients for example like BitTorrent they introduced deep support and all clients have pretty much implemented because it was useful and then you added like a private field to have private trackers and local peer discovery and these types of things and it's like enhancements but it's up to each client how you want to how do you want to deal with it and most clients will likely expose it in some way if you want to use THT and then there's all these considerations and so on. So that's  that in itself could be a local decision for the Status app and it would be separate from the protocol. So the idea is to think of them as two sort of separate animals kind of. That'll be a help in any insights or or thoughts you would like to add to this.
+
+Jacek: Nothing new really. I mean we've talked before about make enshrining the protocol into a separate entity so that when you discuss a protocol change, you do so not from a Unicode perspective and not with a code mentality but rather with a protocol mentality. And I think I think we already agreed that moving to moving the protocol out from the codebase completely including all these little quirks would be the first step and then developing the protocol based on that document obviously with the reference implementation and so on. But like approaching it with this slightly more procedural approach which includes these various steps that that the teams and so on have and tips and whatever, gives it the appropriate social way so that everybody's on the same page and it also gives the community an opportunity to weigh in especially if if you consider that they might not be fluent in you know both called JavaScript, Clojure and in the rest of the technology stack that we happen to be using.
```

Comment:
```suggestion
Jacek: Nothing new really. I mean we've talked before about make enshrining the protocol into a separate entity so that when you discuss a protocol change, you do so not from a Unicode perspective and not with a code mentality but rather with a protocol mentality. And I think I think we already agreed that moving the protocol out from the codebase completely including all these little quirks would be the first step and then developing the protocol based on that document obviously with the reference implementation and so on. But like approaching it with this slightly more procedural approach which includes these various steps that the teams and so on have and tips and whatever, gives it the appropriate social way so that everybody's on the same page and it also gives the community an opportunity to weigh in especially if you consider that they might not be fluent in you know both called JavaScript, Clojure and in the rest of the technology stack that we happen to be using.
```